nodeup/pkg/bootstrap/install.go

```diff
@@ -178,6 +178,6 @@ func (i *Installation) buildSystemdJob() *nodetasks.InstallService {
 	}}
```

Comment:
Probably better placed here:
```suggestion
		Enabled:    fi.PtrTo(false),
	}}
```